docs/rules/require-expect.md

```diff
@@ -0,0 +1,123 @@
+# Ensure that `expect` is called
+
+QUnit's `assert.expect(...)` helps developers create tests that correctly fail
+when their expected number of assertions are not called. QUnit will throw an
+error if no assertions are called by the time the test ends unless a developer
+also calls `assert.expect(0)`. QUnit also has a [configuration option to require
+`expect` for every test](https://api.qunitjs.com/QUnit.config/).
+
+This rule checks for `expect` at linting time. The default "always" option
+requires that `expect` is called in each test. The "except-simple" option only
+requires an `expect` call when an assertion is called inside of a block or when
+`assert` is passed to another function. The rational here is that by wrapping
+`assert` statements with logic, developers are at risk of creating tests that
```

Comment:
Not sure what "with logic" means here. Maybe "by wrapping `assert` statements in conditional blocks or callbacks"?